doc/source/whatsnew/v0.20.0.txt

```diff
@@ -1705,6 +1705,7 @@ Reshaping
 - Bug in ``pd.concat()`` in which concatenating with an empty dataframe with ``join='inner'`` was being improperly handled (:issue:`15328`)
 - Bug with ``sort=True`` in ``DataFrame.join`` and ``pd.merge`` when joining on indexes (:issue:`15582`)
 - Bug in ``DataFrame.nsmallest`` and ``DataFrame.nlargest`` where identical values resulted in duplicated rows (:issue:`15297`)
+- Bug in ``pivot._add_margins` when ``margins_name`` is Unicode under Py2 (:issue:`13292`)
```

Comment:
write this from a user perspective

``Bug in :func:`pandas.pivot_table` incorrectly raising when passing unicode input for ``margins`` keyword``